extension/background/intentRunner.js

```diff
@@ -531,6 +531,42 @@ export function getRegisteredNicknames() {
   return registeredNicknames;
 }
 
+const registeredPageName = {};
+
+export function registerPageName(name, context) {
+  name = name.toLowerCase();
+  if (!context) {
+    delete registeredPageName[name];
+    log.info("Removed nickname for page", name);
+  } else {
+    registeredPageName[name] = context;
```

Comment:
`name` should probably be lower-cased here (i.e., `name.toLowerCase()`)

Reply:
It is already lower-cased on line 551